src/Footer/Footer.tsx

```diff
@@ -0,0 +1,9 @@
+﻿import React from 'react';
+import "./Footer.css";
+export function Footer(){
+    return (
+        <footer id="footer">
```

Comment:
What is the 'id' for here? We don't seem to be using it at the moment?

At the moment the CSS applies to ALL footer elements.
That's probably fine, but an even better option would be to have

```html
<footer class="footer"/>
```

and then 

```css
.footer {
    // styles
}
```

Reply:
actually I see! its used in the tests! (the point about the css still applies though)

For the tests, could you use the `test-id` feature instead?

ie, could we have JSX like this
```jsx
<footer data-testid="footer">
```

and then in the test we can have
```js
getByTestId("footer");
```